billthebuilder/meetup-query
# #163268795 Fix code style to adhere to pep8 standards

#### What does this PR do?
* It fixes the code style to adhere to pep8 standards
#### Description of Task to be completed?
* The enforcement of pep8 standards using `autopep8`
#### How should this be manually tested?
* Clone the repository
* Checkout to the `bg-fix-pep8-163268795` branch
* Create a virtual environment:`virtualenv venv`
* Activate the virtual environment: `source/venv/bin/activate`
* Install the dependencies: `pip install -r requirements.txt`
* Check if code adhere to pep8 guidelines
#### What are the relevant pivotal tracker stories?
* [#163268795](https://www.pivotaltracker.com/story/show/163268795 )
